src/main/java/com/adamk33n3r/runelite/watchdog/WatchdogPanel.java

```diff
@@ -269,12 +269,19 @@ private void createAlert(TriggerType triggerType) {
 
     private PluginPanel createPluginPanel(Alert alert) {
         if (alert instanceof ChatAlert) {
-            ChatAlert chatAlert = (ChatAlert) alert;
+            ChatAlert gameMessageAlert = (ChatAlert) alert;
             return AlertPanel.create(this.muxer, alert)
                 .addAlertDefaults(alert)
-                .addRegexMatcher(chatAlert, "Enter the message to trigger on...", "The message to trigger on. Supports glob (*)")
+                .addRegexMatcher(gameMessageAlert, "Enter the message to trigger on...", "The message to trigger on. Supports glob (*)")
                 .addLabel("<html><i>Note: Will not trigger on<br>player chat messages</i></html>")
                 .build();
+        } else if (alert instanceof PlayerChatAlert) {
+            PlayerChatAlert gameMessageAlert = (PlayerChatAlert) alert;
```

Comment:
Variable name looks to be copy/pasted from above, should be changed to reflect the new type :)

Reply:
Dang it! Good catch. Will fix